saml2/saml2-service-provider/src/opensaml4Main/java/org/springframework/security/saml2/provider/service/web/authentication/OpenSaml4AuthenticationRequestResolver.java

```diff
@@ -69,6 +70,17 @@ public void setAuthnRequestCustomizer(Consumer<AuthnRequestContext> contextConsu
 		this.contextConsumer = contextConsumer;
 	}
 
+	/**
+	 * Set the {@link RequestMatcher} to use for setting the
+	 * {@link OpenSamlAuthenticationRequestResolver#setRequestMatcher(RequestMatcher)}
+	 * (RequestMatcher)}
+	 * @param requestMatcher the {@link RequestMatcher} to identify authentication
+	 * requests
+	 */
```

Comment:
I think this method can be backported to 5.8. As such, will you please add `@since 5.8` to the JavaDoc? After this PR is merged, I will take care of the backporting.